Examples/Scripts/Python/gaussian_sum_filter.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+from pathlib import Path
+from typing import Optional, Union
+
+from acts.examples import Sequencer, GenericDetector, RootParticleReader
+
+import acts
+
+from acts import UnitConstants as u
+
+
+def runCKFTracks(
```

Comment:
Following the `addXXX`/`runXXX` pattern, you could add a `addGSFTracks` function that only configures the GSF, and then a `runGSFTracks` function which configures and example chain (and would be used in a test)